cmd/titus-vpc-service/main.go

```diff
@@ -290,6 +292,7 @@ func main() {
 	rootCmd.Flags().String(sslCertFlagName, "", "The SSL Certificate")
 	rootCmd.Flags().String(sslCAFlagName, "", "General SSL CA")
 	rootCmd.Flags().String(sslTitusAgentCAFlagName, "", "Titus Agent CA")
+	rootCmd.Flags().String(sslValidCNRegexFlagName, `titusagent\..*`, "A regex representing valid CNs/SANS for incoming client certs")
```

Comment:
Not sure if there's a different regex you were going to use rather than this default, but keep in mind that with this default anyone can deploy their own app with stack=titusagent to pass this regex since `${stack}.${appname}....` is one of the SANs the web certs gets.

Reply:
Heh, I was actually aware of that when I wrote it, I just figured it was close enough